scram/route_manager/tests/test_authorization.py

```diff
@@ -125,7 +125,7 @@ def test_unauthorized_after_group_removal(self):
         self.assertEqual(response.status_code, 302)
 
 
-class OidcTest(TestCase):
+class ESnetAuthBackendTest(TestCase):
```

Comment:
Do we have a test already for local auth? If not, can we add one?

Reply:
I couldn't think of a way to check that without just being a test of Django's built in auth which seems like a bad path.

Reply:
Do we still have the code that creates an Admin user and displays the password the first time you connect? We should probably test that, at least.

Reply:
Looking back at the tests, I think [`AuthzTest`](https://github.com/esnet-security/SCRAM/blob/7ca1132942b43a418a59058ca62829a191c6c889/scram/route_manager/tests/test_authorization.py#L12) in `test_authorization.py` covers testing local auth? I also see it testing that admin has permissions. Looking at coverage it seems like we're not testing the autocreation of the admin. I'm not entirely sure how to test that either, but I will think on it and give it a try.

Reply:
Ok tests created for the `AUTOCREATE_ADMIN` part of the home page view. I think now that means we test:

- [x] local authz - For when we use local accounts. AuthzTest Test Case in `test_authorization.py`
- [x] ESnetAuthBackend - For when we use OIDC. ESnetAuthBackend Test Case in `test_authorization.py`
- [x] autocreate_admin - For when we autocreate our admin user (both AUTH_METHODS). `test_autocreate_admin.py`